### PR TITLE
Explain that LXD containers run Docker containers inside

### PR DESCRIPTION
**Pull request by techtonik**
Fixes issue #3612 

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3660)

### CODE REVIEW

**Comment by marcoceppi** [comment]
:+1: LGTM

File: docs/getting-started-guides/ubuntu/local.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ title: Local Kubernetes development with LXD
...
+Running Kubernetes locally has obvious development advantages, such as lower cost and faster iteration than constantly deploying and tearing down clusters on a public cloud. Ideally a Kubernetes developer can spawn all necessary nodes inside local containers and test new configurations as they are committed. This page will show you how to deploy a cluster to LXD containers on a local machine.
 {% endcapture %}
 
-The purpose of using [LXD](https://linuxcontainers.org/lxd/) on a local machine is to emulate the same deployment that a user would use in a cloud or bare metal. Each node is treated as a machine, with the same characteristics as production.
+The purpose of using [LXD](https://linuxcontainers.org/lxd/) on a local machine is to emulate the same deployment that a user would use in a cloud or bare metal. Each node is treated as a machine, with the same characteristics as production. Each node is a separate container, which runs `Docker` containers and `kubectl` inside (see [Cluster Intro](https://kubernetes.io/docs/tutorials/kubernetes-basics/cluster-intro/) for more info).
```
Comment (ahmetb) [inline docs/getting-started-guides/ubuntu/local.md]: No need to do `Docker`, just Docker is fine. other than that, LGTM.

**Comment by techtonik** [comment]
No missing articles this time. Awesome. Thanks for review. =)

**Comment by chenopis** [comment]
@techtonik I made some small changes directly. =)

**Comment by techtonik** [comment]
Oh, no. I will never get them. )

**Pull request merged**